File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size() != t.size()) return false;
        vector<int> arr(26,0);
        for(char ch : s) arr[ch - 'a']++;
        for(char ch : t) arr[ch - 'a']--;
        for(int i=0; i<26; i++) {
            if(arr[i] != 0) return false;
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()) return false;
        int v[] = new int[26];
        for(char ch : s.toCharArray()) v[ch - 'a']++;
        for(char ch : t.toCharArray()) v[ch - 'a']--;
        for(int num : v) {
            if(num != 0) return false;
        }
        return true;
    }
}